lib/linux.js

```diff
@@ -1,7 +1,10 @@
 'use strict';
 const path = require('path');
+
 const childProcess = require('child_process');
+
 const pify = require('pify');
+
```

Comment:
It should be like this

```js
const path = require('path');
const childProcess = require('child_process');
const pify = require('pify');

const cp = pify(childProcess);
```

Same goes for `win.js` and `macos.js`. A newline has to be added after the `require` statements at [win.js#L5](https://github.com/sindresorhus/wallpaper/blob/master/lib/win.js#L5) and [macos.js#L5](https://github.com/sindresorhus/wallpaper/blob/master/lib/macos.js#L5). That will fix this linting issues with travis.